crates/evm/src/tests/queries/evm_call_tests.rs

```diff
@@ -96,7 +97,7 @@ fn test_state_change() {
             value: Some(U256::from(123134235)),
             ..Default::default()
         },
-        Some(BlockNumberOrTag::Latest),
+        Some(BlockId::Number(BlockNumberOrTag::Latest)),
```

Comment:
This would be `None` in this case, no?

Reply:
Yeah None is interpreted the same as latest in most queries